tests/tests/test_api.py

```diff
@@ -4,14 +4,23 @@
 from django.test import TestCase
 from wagtail.core.models import Page
 
+from wagtail_transfer.auth import digest_for_source
 from tests.models import PageWithRichText, SectionedPage
 
 
 class TestPagesApi(TestCase):
     fixtures = ['test.json']
 
+    def get(self, page_id):
+        digest = digest_for_source('local', str(page_id))
+        return self.client.get('/wagtail-transfer/api/pages/%d/?digest=%s' % (page_id, digest))
```

Comment:
I think we should pass this token using the `Authorization` header and use a 401 response code for errors.

Reply:
I suspect that could get messy if sites have their own auth layer in place, e.g. staging site behind basic auth. This way, our mechanism is separate from anything that exists at the server level.